dealii/fulldepsspack/Dockerfile

```diff
@@ -4,8 +4,12 @@ MAINTAINER luca.heltai@gmail.com
 
 # get deal.II repo
 USER       root
+
 ARG 	   VER=develop
-RUN        spack install -j12 dealii@$VER \
+RUN        spack install dealii@$VER \
            && spack clean -a
 
+RUN        echo "spack load dealii ninja numdiff cmake openmpi" \
+           >> /etc/profile.d/spack.sh
```

Comment:
Do I have to manually source this file? At least for the current build I need to do
```
docker run --rm -it dealii/dealii:9.0.0-gcc-mpi-fulldepsspack-debugrelease
dealii@f2f787110cff:~$ cmake
bash: cmake: command not found
dealii@f2f787110cff:~$ source /etc/profile.d/spack.sh
dealii@f2f787110cff:~~$ cmake --version
cmake version 3.11.1
```

Reply:
I always run the image adding "bash" at the command line. I guess we should change the entry point of the image, so that it actually runs bash AND it sources profile files